docker/local-dev/Dockerfile

```diff
@@ -43,6 +43,10 @@ RUN apt-get install -y \
         python3-dev \
         vim
 
+# Upgrade pip to be compatible with new platform tag formats
+RUN pip install --upgrade pip
+RUN pip3 install --upgrade pip
```

Comment:
Unsure if you'd prefer to pin these, let me know.

Reply:
I don't think we need to pin them.